tasks/release.py

```diff
@@ -1266,3 +1267,43 @@ def update_current_milestone(ctx, major_version: int = 7, upstream="origin"):
             milestone_branch,
             next,
         )
+
+
+def send_slack_msg(ctx, channel_id, message, webhook=None):
+    webhook = webhook or os.environ.get("SLACK_DATADOG_AGENT_CI_WEBHOOK")
+    payload = {
+        'channel_id': channel_id,
+        'message': message,
+    }
+    ctx.run(f'curl -X POST -H "Content-Type: application/json" --data "{payload}" {webhook}')
+
+
+@task
+def check_previous_agent6_rc(ctx):
+    """
+    Validates that there are no existing Agent 6 release candidate pull requests
+    and checks if an Agent 6 build pipeline has been run in the past week
+    """
+    err_msg = ""
+    agent6_prs = ""
+    github = GithubAPI()
+    prs = github.get_pr_for_branch(None, "6.53.x")
+    for pr in prs:
+        if "Update release.json and Go modules for 6.53" in pr.title and not pr.draft:
+            agent6_prs += f"\n- {pr.title}: https://github.com/DataDog/datadog-agent/pull/{pr.number}"
+    if agent6_prs:
+        err_msg += "AGENT 6 ERROR: The following Agent 6 release candidate PRs already exist. Please address these PRs before creating a new release candidate"
+        err_msg += agent6_prs
+
+    response = get_ci_pipeline_events(
+        'ci_level:pipeline @ci.pipeline.name:"DataDog/datadog-agent" @git.tag:6.53.* -@ci.pipeline.downstream:true',
+        7,
+    )
+    if not response.data:
+        err_msg += "\nAGENT 6 ERROR: No Agent 6 build pipelines have run in the past week. Please trigger a build pipeline for the next agent 6 release candidate."
+
+    if err_msg:
+        print(err_msg)
+        # send slack message to #agent-ci-on-call channel
+        send_slack_msg(ctx, "C0701E5KYSX", err_msg)
+        sys.exit(1)
```

Comment:
```suggestion
        # send slack message to #agent-ci-on-call channel
        send_slack_msg(ctx, "C0701E5KYSX", err_msg)
        raise Exit(message=err_msg, code=1)
```
We usually use the invoke Exit